callstack/react-native-paper
# fix: add bg color to list accordion to limit ripple area

Fixes: https://github.com/callstack/react-native-paper/issues/2683

### Summary

It turns out there is need to wrap `TouchableRipple` within `View` with specified background color to prevent ripple effect from spreading outside the component area. 

NOTE: the similar approach is applied in Button component, however `Surface` component is used there, which we don't need there. 



### Test plan

1. Run the example app on Android 7
2. Go to list component
3. Press the list item and observe the ripple
